ProjectManagmentSystem/src/Modules/AuthenticationModule/components/ResetPass/ResetPass.tsx

```diff
@@ -1,7 +1,147 @@
-import React from 'react'
 
+import logo from '../../../../Modules/../assets/images/PMS 3.svg'
+import { TextField } from '@mui/material';
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 export default function ResetPass() {
+  let  navigate = useNavigate()
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const onSubmit =  async (data) => {
+    console.log(data)
```

Comment:
We should remove console logs before merging the branch to master to avoid revealing any sensitive data to users.